FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -150,6 +165,28 @@ public static void ValidateConfiguration(this IServiceCollection services)
             }
         }
 
+        // agent name is used as part of table name and queue name, need to validate agent name to contain only alphanumeric characters, and not begin with a numeric character.
+        // Reference: https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-the-table-service-data-model#table-names
+        // https://docs.microsoft.com/en-us/rest/api/storageservices/naming-queues-and-metadata#queue-names
```

Comment:
I saw that valid "queue name must be from 3 through 63 characters long" from the doc, maybe add a length validate?

Reply:
Added it.